src/server.h

```diff
@@ -2068,6 +2068,10 @@ struct valkeyServer {
     unsigned long long cluster_link_msg_queue_limit_bytes; /* Memory usage limit on individual link msg queue */
     int cluster_drop_packet_filter;                        /* Debug config that allows tactically
                                                             * dropping packets of a specific type */
+
+    uint32_t
+        debug_cluster_close_link_on_packet_drop : 1; /* Debug config that goes along with cluster_drop_packet_filter.
+                                                        When set, the link is closed on packet drop. */
```

Comment:
Did clang format do this automatically? This is such a weird format to split the type and the variable name on to to different lines.

Reply:
Yes, clang did it. Agree..

Reply:
Yeah, this is why it's good that we don't run clang-format on merge. Now at least we have a chance to fix it manually. Just move the comment to it's own line.

```suggestion
    /* Debug config that goes along with cluster_drop_packet_filter.
     * When set, the link is closed on packet drop. */
    uint32_t debug_cluster_close_link_on_packet_drop : 1;
```

Reply:
Btw, using one bit for this field will not save memory. We'd need to do it for some more fields and put them next to each other. Otherwise IMHO there's not point.

Reply:
Agree, there is no value now, but by keeping it to a bit, we can keep the intent clear and do a subsequent merge of such fields like you pointed out in the other comment (https://github.com/valkey-io/valkey/pull/461#discussion_r1612056999).

Reply:
OK. You're welcome to include this refactoring in this PR, but I won't insist. We'll open a follow-up issue for it otherwise.

Reply:
> We'll open a follow-up issue for it otherwise.

We do have a hard problem to solve which is that the config system (as it is written) doesn't support bit fields since bit fields are not addressable. Most of the booleans we have are bit fields.

Reply:
Started a topic here: https://github.com/valkey-io/valkey/issues/590